generated/nidaqmx/system/device.py

```diff
@@ -130,6 +131,52 @@ def do_ports(self):
 
     # endregion
 
+    # region Calibration Info property
+
+    @property
+    def ext_cal_last_date_and_time(self):
+        """
+        Indicates the last date and time that the device underwent an
```

Comment:
Add the property type.

```suggestion
        datetime: Indicates the last date and time that the device underwent an
```